components/UserMenu.js

```diff
@@ -62,12 +67,17 @@ const ItemLink = styled.a`
 
 const UserMenu = (props) => {
   const { username = '' } = props
+  const handleLogout = () => {
+    console.log('logging out')
+  }
   return (
     <UserMenuContainer>
       <MenuContainer>
         {
           !username ?
-            <LoginButton>Login</LoginButton> :
+            <Link href="/login">
```

Comment:
I think that we should mantain the Login button without the Link because we are going to use a Modal instead of another page.

Reply:
I undestand, this logic doesn't apply to the router